package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -445,8 +445,8 @@ def __init__(self, filename, n_atoms=None, **kwargs):
         self.trjfile = scipy.io.netcdf.netcdf_file(self.filename,
                                                    mmap=self._mmap)
 
-        if not ('AMBER' in self.trjfile.Conventions.split(',') or
-                'AMBER' in self.trjfile.Conventions.split()):
+        if not ('AMBER' in self.trjfile.Conventions.decode('utf-8').split(',') or
+                'AMBER' in self.trjfile.Conventions.decode('utf-8').split()):
             errmsg = ("NCDF trajectory {0} does not conform to AMBER "
```

Comment:
I got an additional related failure in `test_netcdf`:
```
[gw3] darwin -- Python 3.6.1 /Users/oliver/anaconda3/envs/mda/bin/python

self = <testsuite.MDAnalysisTests.coordinates.test_netcdf.TestNCDFReader testMethod=test_metadata>

    def test_metadata(self):
        data = self.universe.trajectory.trjfile
>       assert_equal(data.Conventions, 'AMBER')
E       AssertionError:
E       Items are not equal:
E        ACTUAL: b'AMBER'
E        DESIRED: 'AMBER'
```

The test also needs a `decode('utf-8')`.